src/pybind/mgr/cephadm/module.py

```diff
@@ -2271,7 +2273,9 @@ def _create_daemon(self, daemon_type, daemon_id, host,
             cephadm_config = self._get_config_and_keyring(
                     daemon_type, daemon_id,
                     keyring=keyring,
-                    extra_config=extra_config)
+                    extra_ceph_config=extra_config.get('config'))
```

Comment:
`extra_config` is actually a `str

Reply:
ahh yeah, looks like a rebase issue, cause that definitely doesn't look right.

Nice spot.